tools/release/promote_to_rc.ts

```diff
@@ -146,6 +165,8 @@ async function promoteBinaryToRc(binary: string, target: string) {
     rcBinaryName,
   );
   await runPatchver(unzippedName, target, rcBinaryName);
+  await runRcodesign(target, rcBinaryName);
```

Comment:
Oh I forgot this: only sign the `deno` binary. `denort` is strippped of signatures anyways during `deno compile`